.github/ISSUE_TEMPLATE/Patch-Version-Release-Checklist.md

```diff
@@ -0,0 +1,135 @@
+
+- Release Manager: <!-- Insert Release Manager Here -->
+- Release Date: <!-- Insert Release Date Here -->
+- Release Branch: `main`
+
+# Release Steps
+The release manager is expected to execute the checklist below. The release manager is also responsible for ensuring that the checklist is updated as the release progresses. Any changes or issues should be communicated under this issue for centralized tracking.
+
+## 1. Codebase Stability
+- [ ] The `main` branch build passes
+
+    [![Build Status](https://github.com/celery/celery/actions/workflows/python-package.yml/badge.svg)](https://github.com/celery/celery/actions/workflows/python-package.yml)
+
+## 2. Breaking Changes Validation
+A patch release should not contain any breaking changes. The release manager is responsible for reviewing all of the merged PRs since the last release to ensure that there are no breaking changes. If there are any breaking changes, the release manager should discuss with the maintainers to determine the best course of action if an obvious solution is not apparent.
+
+## 3. Compile Changelog
+The release changelog is set in two different places:
+1. The [Changelog.rst](https://github.com/celery/celery/blob/main/Changelog.rst) that uses the RST format.
+2. The GitHub Release auto-generated changelog that uses the Markdown format. This is auto-generated by the GitHub Draft Release UI.
+
+> ⚠️ **Warning:** The pre-commit changes should not be included in the changelog.
+
+To generate the changelog automatically, [draft a new release](https://github.com/celery/celery/releases/new) on GitHub using a fake new version tag for the automatic changelog generation. Notice the actual tag creation is done **on publish** so we can use that to generate the changelog and then delete the draft release without publishing it thus avoiding creating a new tag.
+
+- Create a new tag
+<img width="502" alt="CleanShot 2023-09-05 at 22 06 24@2x" src="https://github.com/celery/celery/assets/4662342/69d7eb78-0dd1-4a37-9266-d718e2340fad">
+
+- Generate Markdown release notes
+<img width="389" alt="CleanShot 2023-09-05 at 22 13 39@2x" src="https://github.com/celery/celery/assets/4662342/83c6a4b6-1676-496e-8d75-5a6b10352b5b">
+
+- Copy the generated release notes.
+
+- Delete the draft release without publishing it.
+
+### 3.1 Changelog.rst
+Once you have the actual changes, you need to convert it to rst format and add it to the [Changelog.rst](https://github.com/celery/celery/blob/main/Changelog.rst) file. The new version block needs to follow the following format:
+```rst
+.. _version-x.y.z:
+
+x.y.z
+=====
+
+:release-date: YYYY-MM-DD HH:MM P.M/A.M TimeZone
+:release-by: Release Manager Name
+
+Changes list in RST format.
```

Comment:
we can consider changing the changelog from rst to md in near future. this will reduce some repeating/converting stuff

Reply:
We cannot change the changelog format to markdown since we include the changelog in our documentation site which uses RST.

Reply:
wasn't fully aware of that. have to cross check. thanks for sharing the insight

Reply:
> We cannot change the changelog format to markdown since we include the changelog in our documentation site which uses RST.

Few lines below here:
```
These changes will reflect in the [Change history](https://docs.celeryq.dev/en/stable/changelog.html
```

I think that's what @thedrow was referring to @auvipy - I added it specifically for that Markdown/RST formatting question/topic.